src/aura/STG_CMP_Affl/STG_CMP_Affl.cmp

```diff
@@ -393,7 +394,11 @@
 
                         <ui:button aura:id="newAfflMappingBtn" class="slds-button slds-button--brand slds-m-top--medium affl-mapping-addnew" press="{!c.newAfflMapping}" disabled="true">
                             <ui:outputText value="{!$Label.c.stgBtnAddMapping}" />
-                        </ui:button>
+                        </ui:button>                       
+                        <br/>
+                        <aura:if isTrue="{!v.showAfflError}">
```

Comment:
This behaves as it should, but I think it would be better to follow SLDS' conventions and display an error toast instead: https://lightningdesignsystem.com/components/toast/#Error.

Reply:
I think that is how it works for "Run Clean Up" do we still want to do a toast error instead?

Reply:
@carlos -- one of the reasons we want to revisit settings is that by us embedding this in visualforce we've run into issues in the past with being able to throw said toasts. What I wound up doing when we had a request for a toast on save settings was creating a hidden div to hide/show on error. We can do that, but at some point, if we want to throw real toasts, we need to either drop our marriage to the VF page, or "componentize" our version of a toast for re-usage, or wait until we rewrite settings.
For now if we want a toast we can copy/pasta -- if we are going to hopefully rewrite settings, I'd rather not invest too much time into this UI. Open to chat though.
cc: @skosgi1

Reply:
Well, the Run Cleanup message is not really an error or a warning. So it's ok is less prominent. Also, I see something that looks more like a toast in one of the pages:
<img width="1366" alt="Screen Shot 2019-11-01 at 5 28 50 PM" src="https://user-images.githubusercontent.com/1894238/68060187-5bca7180-fccd-11e9-9758-d7c552b3ca34.png">

Reply:
Looks like above one is not a toast message, I have now included a toast message for Affiliations.